Spanish interview (#406)
ropensci/roweb3: content/blog/2022-11-22-r-universe-stars-1-en/index.md

---
slug: "r-universe-stars-1-en"

> title: R Community,Exchange and Learn

Author: I like this.

author:
  - Yanina Bellini Saibene

>   - Alejandra Bellini

Reviewer: Do you think Alejandra and Lucio would like to create and connect GitHub accounts so they have nice profile pictures?

Author: Do we take the pictures from GitHub?.  I add two pictures for them on the author profile. Perhaps I should name it avatar.jpg right?  They are not developers so I don't think they will have to have a GitHub profile.

Author: Maëlle explain a way to add pictures.

  - Lucio Casalla  
  - Steffi LaZerte
date: '2022-11-22'
tags:
  - community
  - r-universe
# The summary below will be used by e.g. Twitter cards
description: "This is the first post of our interview series __\"Meeting the stars of the R-universe\"__. We begin our journey in _Argentina_ with a team that uses R and develops R packages in the Argentinean State."
---

> [Artículo en español](/blog/2022/11/22/r-universe-stars-1-es/)

Author: I didn't see the purply-blue box but I would prefer that one instead of the quote. Can yiu change it ok both blog post?

This is the first post of our interview series __"Meeting the stars of the R-universe"__. We begin our journey in _Argentina_  with a team that uses R and develops R packages in the Argentinean State.

## The experience of working with R packages in the Argentine state

> 

Author: Yes! So much better.  Thanks!

At this stage, the objective was achieved: the team learn the tool and the work dynamics that they had defined as the most appropriate.

The second step was to _"open the process"_ as well as _"improve the workflow"_. Pablo explains that they also incorporated the _"dynamics of an open State, open our process, not hiding how we do our work."_. A concept that in the future will also include the possibility of receiving contributions from the community and or our code to be reviewed.

> As all processes are complex and many of them involve simultaneous and even "chaotic" steps, as _Tucu_ describes, where different people with diverse knowledge contributed to the design of the projects that are currently under development.

Reviewer: ```suggestion
All processes are complex and many of them involve simultaneous and even "chaotic" steps, as _Tucu_ describes. These involve different people with diverse knowledge contributing to the design of the projects that are currently under development.
```

> _Tucu_ gives as an example the _Thursday meeting_ where they show and share their codes so that other people can use them in other parts of the process: _"We do live coding where we raise doubts and do exercises (...) we try to generate a dynamic in which we share in a ludic way, sometimes showing each part and what they are doing to be shared with others."_

Reviewer: "playful" might be a bit more accessible than "ludic" (I've never encountered that word before! Nice to learn something new :grin:)

```suggestion
To integrate these different elements, _Tucu_ gives as an example the _Thursday meeting_. There, they show and share their code so that others can use them in different parts of the process. _"We do live coding where we raise doubts and do exercises (...) we try to generate a dynamic in which we share in a playful way, sometimes showing each part and what they are doing to be shared with others."_
```

## Free software and in our language

In addition to the free nature of open source software and the importance of resource management for a State, Pablo Tiscornia explains what appears almost as a declaration of principles: _“It is a work philosophy, and an idea about how we develop and promote knowledge. I think what we can give, today to the State, is thanks to free software. So, it would be almost irresponsible not to promote what formed us.”_

> Juan Pablo reinforces the idea with his personal experience: _“I was a master's thesis student in political science and seven years ago I forced myself to do my thesis in R in order to learn (...) and what I quickly found, luckily, was the R community.”._

Reviewer: Awwwww! ❤️ 
```suggestion
Juan Pablo reinforces the idea with his personal experience: _“I was a master's thesis student in political science and seven years ago I forced myself to do my thesis in R in order to learn (...) and what I quickly found, luckily, was the R community.”_
```

> 

Reviewer: Perhaps add this awesome quote as a 'big' quote? Or too much?

```suggestion
> "and what I quickly found, luckily, was the R community"
```

> __Exchanging with the community and learning from the community seems to be the main mantra of the R community that both Tucu and Pablo share with the team.__

Reviewer: ```suggestion
__"Exchange with the R community and learn from the R community" seems to be the main mantra that both _Tucu_ and Pablo share with the team.__
```

## A work for the community

> The audiences that receive the work developed in the office are multiple.

Reviewer: ```suggestion
The work developed by this office serves a many different audiences.
```

> {{< figure src = "sinta.png" class = "pull-left" width = "380" alt = "SINTA system home page with access to its open datasets." >}}

Reviewer: ```suggestion
{{< figure src = "sinta.png" class = "pull-left" width = "380" alt = "Screenshot of the SINTA system home page showing access to its open datasets." >}}
```

> Today they have a site called [SINTA (Sistema de Información Turístico de la Argentina)](https://www.yvera.tur.ar/sinta/) where they present in an orderly fashion the _"different ways of accessing information"_. Juan Pablo explains that _"the work was to develop tools. R has much to do with this, bringing the information closer to different audiences"_.

Reviewer: ```suggestion
Today they have a site called [SINTA (Sistema de Información Turístico de la Argentina)](https://www.yvera.tur.ar/sinta/) where the _"different ways of accessing information"_ are present in an organized fashion. Juan Pablo explains that _"the work was to develop tools. R has much to do with this, bringing the information closer to different audiences."_
```

> Those audiences include academics (people who research and teach) and government. So, for example, different offices at different levels of the State need the data, and the office they work for that uses the information for decision-making and public policy setting. Also, all the people that want to interact with the information in the open data portal.

Reviewer: ```suggestion
Those audiences include academics (people who research and teach) and government. For example, offices at different levels of the State can use the information for decision-making and setting public policy. As this is an open data portal, the information is also available to anyone else who wants to interact with it.
```

</br>

> ## The R-universe experience

Reviewer: ```suggestion
## The R-Universe experience
```

> The concept of community is closely related to social and communication networks, and it was precisely Twitter that first brought Juan Pablo closer to [R-universe](https://r-universe.dev/search/) where he discovered [the step-by-step on how to build his own R universe.](/blog/2021/09/21/creando-tu-r-universe/).

Reviewer: the?
```suggestion
The concept of community is closely related to social and communication networks, and it was precisely Twitter that first brought Juan Pablo closer to the [R-Universe](https://r-universe.dev/search/) where he discovered [step-by-step instructions on how to build his own R universe](/blog/2021/09/21/creando-tu-r-universe/).
```

> _“It is the logical next place to share the things that we're doing openly, to make the packages in our GitHub available. It is possible to open up our little marketplace,our mini CRAN, in R-Universe, to share the libraries that we're starting to develop.”_, this is how the _Tucu_ explains the first use of the platform and adds: _“to me it looks spectacular, as I see how it is changing over time, how comprehensive it is, the way one can visualize all the development that one is sharing in the R-Universe”_.

Reviewer: ```suggestion
_“It is the logical next place to share the things that we're doing openly, to make the packages in our GitHub available. It is possible to open up our little marketplace, our mini CRAN, in R-Universe, to share the libraries that we're starting to develop”_, this is how _Tucu_ explains the first use of the platform and adds, _“to me it looks spectacular, as I see how it is changing over time, how comprehensive it is, the way one can visualize all the development that one is sharing in the R-Universe.”_
```

> Pablo adds that developing packages was the logical next step of the process after the training and the incorporation of the R language in the office: _“we put together, on the one hand, the toolswith which we improve our workflow, the package {Herramientas}, and on the other hand, the package {Comunicación}, that the idea was mainly, and still is today, mainly to improve the workflow of our team, our day-to-day work.”_.

Reviewer: ```suggestion
Pablo adds that developing packages was the logical next step of the process after incorporating the R language and training: _“we put together, on the one hand, the tools with which we improve our workflow, the package {Herramientas}, and on the other hand, the package {Comunicación}, that the idea was mainly, and still is today, mainly to improve the workflow of our team, our day-to-day work.”_
```

> They also add two important objectives to the whole process: to smooth the language and make it accessible to all people, even those who do not necessarily come from programming backgrounds, and to _"develop the packages in Spanish as much as possible”_. The target was the same: _We couldn't get into thinking whether we know English, if we don't know English, or even how much English we know. We wanted to get that barrier out of the way._

Reviewer: I'm not sure what "smooth the language" means. Do you mean "clarify and annotate the code", or language as in Spanish?

Also, I would suggest moving this to the above section "Free software and in our language", as that section doesn't actually mention making the packages in Spanish, but I think it should (?)

```suggestion
They also add two important objectives to the whole process: to smooth the language and make it accessible to all people, even those who do not necessarily come from programming backgrounds, and to _"develop the packages in Spanish as much as possible”_. This helped reduce barriers for the team: _"We couldn't get into thinking whether we know English, if we don't know English, or even how much English we know. We wanted to get that barrier out of the way."_
```

Author: When they speak about language there they mean R. I'm thinking in a better word

> {{< figure src = "dnme-r-universe.png" class = "pull-right" width = "500" alt = "r-universe profile with SINTA's packages" >}}

Reviewer: ```suggestion
{{< figure src = "dnme-r-universe.png" class = "pull-right" width = "500" alt = "Screenshot of an R-Universe profile showing SINTA's packages" >}}
```

> Both Pablo and Juan Pablo agree that the main potential of R-universe for the office is _being able to host all the packages they have under development._ Pablo defines it like this: _“So it seems to me that the main use I see today, a priori, is to be a great place where we can put all of our toolboxes and we can even show our packages as well. [We have a place with our tools that are consistent with each other, that has a common philosophy of work](https://dnme-minturdep.r-universe.dev/ui#packages)”_.

Reviewer: ```suggestion
Both Pablo and Juan Pablo agree that the main benefit of the R-Universe to their office is being able to host all the packages they have under development. Pablo describes the R-Universe as _"a great place where we can put all of our toolboxes and we can even show our packages as well. We have a [place](https://dnme-minturdep.r-universe.dev/ui#packages) with our tools that are consistent with each other, that have a common philosophy of work.”_
```

> The work of Juan Pablo and Pablo as well as that of R-universe are inscribed in a world that has changed its structures and as they define it in the talk. In Pablo's words: _“We are in a world where programming is no longer for programmers in terms of who studied for it, we are in a world where there are many people who collaborate and want to collaborate even with only one little piece.”_.

Reviewer: ```suggestion
The work of Juan Pablo and Pablo as well as that of the R-Universe are in a world that has changed how programming is used. In Pablo's words: _“We are in a world where programming is no longer for programmers in terms of who studied for it, we are in a world where there are many people who collaborate and want to collaborate even with only one little piece.”_
```

> Juan Pablo concludes by saying: _“There are more barriers that can be lowered, especially for those of us who do not come from programming, we come from the social sciences and began to program and started using tools that made our work easier. There is an instance to facilitate even more the work that we can write with the tidy philosophy with its manifesto and that "we can also do it in Spanish."_. 

Reviewer: I found that second half a bit confusing and less meaningful. What do you think about splitting it up?

```suggestion
Juan Pablo concludes by saying: _“There are more barriers that can be lowered, especially for those of us who do not come from programming, we come from the social sciences and began to program and started using tools that made our work easier"_  and importantly, _"we can also do it in Spanish."_ 
```

> 

Reviewer: I think it'd be nice to a) have a sentence explaining the interview video and b) be clear that some of what is in the text above isn't necessarily in the video (right? at least some looked new to me...?)

If we use a heading, we can even link to it at the beginning.

```suggestion
### Video Interview 
**Watch excerpts from the interview below!**
```

Author: I added because I didn't see this one. Thanks!
